sw/device/lib/base/status.c

```diff
@@ -117,3 +117,6 @@ bool status_extract(status_t s, const char **code, int32_t *arg, char *mod_id) {
     return false;
   }
 }
+
+extern bool status_ok(status_t s);
+extern absl_status_t status_err(status_t s);
```

Comment:
Does `extern` here override `__attribute__((always_inline)) inline` in the functions' declarations?

Reply:
Discussed offline, the style guide also has a section on this: https://opentitan.org/book/doc/contributing/style_guides/c_cpp_coding_style.html#inline-functions